src/encoded/static/components/summary.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import queryString from 'query-string';
-import _ from 'underscore';
```

Comment:
Is this relevant to solving the ticket? I am not being sarcastic. No. I ask because I did not see how it contributes to solving the ticket and I would like to know just in case I missed it.

Reply:
No, it's not relevant, but I was getting a linting error and wanted to fix it.